Source/Krypton Toolkit/Shared/Krypton.Toolkit.Suite.Extended.Tools/Classes/General/MissingFrameWorkAPIs.cs

```diff
@@ -8,6 +8,8 @@
 
 using System.Runtime.CompilerServices;
 
+using Microsoft.VisualBasic;
```

Comment:
Why pull in the massive dll for 1 functions ?

Reply:
Cleaned it up